plugin/adsk/scripts/mayaUsd_USDRootFileRelative.py

```diff
@@ -181,18 +197,13 @@ def connectToDialogControls(cls, parentLayout):
         Used so we can update the file path preview fields.
         """
 
-        # Get the Qt pointer for the input parent string (layout from mel).
-        ptr = omui.MQtUtil.findLayout(parentLayout)
-        if ptr is not None:
-            # Find the top-level window from that parent layout.
-            maya_widget = wrapInstance(int(ptr), QWidget)
-            pp = maya_widget.parent()
-            while pp:
-                if pp.inherits('QFileDialog'):
-                    cls._fileDialog = pp
-                pp = pp.parent()
-                if pp:
-                    maya_window = pp
+        # Get the Qt Window containing the input parent layout from mel.
+        maya_window_name = usdFileRelative.findWindowNameFromLayout(parentLayout)
+        maya_window = usdFileRelative.findQtWindowFromTitle(maya_window_name)
+        if maya_window:
+            cls._fileDialog = maya_window
+        else:
+            print('Could not find dialog named %s' % maya_window_name)
```

Comment:
What is the purpose of the print? The user cannot really do anything to fix this error right? If you want the user to know about it then maybe use cmds.warning().